commands/apps_dev.go

```diff
@@ -106,33 +109,36 @@ func AppsDev() *Command {
 // RunAppsDevBuild builds an app component locally.
 func RunAppsDevBuild(c *CmdConfig) error {
 	ctx := context.Background()
-	noCache, err := c.Doit.GetBool(c.NS, doctl.ArgNoCache)
-	if err != nil {
-		return err
-	}
-	timeout, err := c.Doit.GetDuration(c.NS, doctl.ArgTimeout)
+
+	conf, err := newAppDevConfig(c)
 	if err != nil {
-		return err
+		return fmt.Errorf("initializing config: %w", err)
 	}
+
+	var (
+		// doctlConfig is the CLI config source.
+		doctlConfig = config.DoctlConfigSource(c.Doit, c.NS)
```

Comment:
Do you think it is worth wrapping this logic inside `newAppDevConfig` and creating convenience methods to access the corresponding config types? I just worry this is a lot of non-obvious boilerplate each run command has to manage.

Reply:
Yep that would be nicer 👍🏼